src/Geno/C++/GUI/Widgets/OutputWindow.cpp

```diff
@@ -141,5 +171,24 @@ void OutputWindow::Capture( void )
 
 		for( size_t bytes_read = 0; bytes_read < BytesInFront; bytes_read += _read( m_Pipe[ READ ], &m_Captured[ OldSize + bytes_read ], ( uint32_t )( BytesInFront - bytes_read ) ) );
 	}
+	#else
+	int64_t StartingOffset = lseek( m_Pipe[ READ ], 0, SEEK_CUR );
+	size_t  BytesInFront   = ( size_t )lseek( m_Pipe[ READ ], 0, SEEK_END );
+	size_t  OldSize        = m_Captured.size();
+
+	lseek( m_Pipe[ READ ], StartingOffset, SEEK_SET );
+
+	try
+	{
+		// TODO: This causes an OOM for some reason
+		m_Captured.resize( OldSize + BytesInFront );
```

Comment:
Have you tried replacing the m_Captured variable with a good-old char* that we malloc/realloc ourselves? in case the libc is pulling a prank on us here